palantirnet/federated-search-demo
# Fs 32 dual servers

## JIRA Ticket(s)

https://palantir.atlassian.net/browse/FS-32

## Description

Adds a `Drupal 8 schema` to all the Search API configurations.
Renames the local_server as `Drupal 7 schema - default`

## Testing instructions

1. Checkout this branch
1. Run `phing install-all`
1. Ensure that the schemas are installed on http://d7.fs-demo.local/admin/config/search/search_api and the corresponding sites.
1. Ensure that you can switch the Federated Search index to use the `Drupal 8 schema` server and still index content.
